assignments/chess-board/code.js

```diff
@@ -38,11 +38,21 @@ let board =
 const row = (y) => Math.floor(y / (height / 8))
 const column = (x) => Math.floor(x / (width / 8))
 
-const select = (x, y) => {
-  drawText(board[row(y)][column(x)], width / 8 * column(x), height / 8 * (row(y) + 1), 'red', 45)
+let aSelect = []
+const move = (x, y) => {
+  if (aSelect !== []) {
+    drawText(aSelect, width / 8 * column(x), height / 8 * (row(y) + 1), 'black', 30)
+    board[row(y)][column(x)] = aSelect[0]
+    aSelect.pop(board[row(y)][column(x)])
```

Comment:
And here you want to clear out `selectedPiece`

```suggestion
    selectedPiece = null
```

(A note on `pop` even though you don't need to use it if you take these suggestions: it doesn't take any arguments. It just always removes the last item in the list, so `aSelect.pop()` is how you'd call it.)